react-graphql-apollo-tutorial.md

```diff
@@ -908,9 +912,9 @@ const errorLink = onError(({ graphQLErrors, networkError }) => {
 });
 {{< /highlight >}}
 
-For instance, you could differentiate the error handling on application level into development and production mode. When developing your application, it might be sufficient to console log the errors to your developer console in the browser. When in production mode, you can setup a error tracking service such as {{% a_blank "Sentry" "https://sentry.io" %}}. That's how you would always see the errors in a web accessible dashboard which would perhaps helps you to address and solve them as bugs.
+You could differentiate the error handling at the application level into development and production mode. During development, it might be sufficient to console log the errors to a developer console in the browser. In production mode, you can setup a error tracking service like {{% a_blank "Sentry" "https://sentry.io" %}}. It will teach you to identify bugs in a web dashboard more efficient
 
-Now you have two links in your application: `httpLink` and `errorLink`. How can you combine both for creating the Apollo Client instance? There exists another useful package in the Apollo ecosystem that makes link compositions possible. First, install it on the command line:
+Now you have two links in your application: `httpLink` and `errorLink`. To combing them for use with the Apollo Client instance, we'll download yet anothe ruseful package in the Apollo ecosystem that makes link compositions possible in the command line:
```

Comment:
"we'll download yet anothe ruseful package" to "we'll download yet another useful package"

Reply:
"we'll download yet anothe ruseful package in the Apollo ecosystem that makes link compositions possible in the command line:" to "we'll install yet another useful package in the command line from the Apollo ecosystem that makes link compositions possible:"